### PR TITLE
Barcelos_module

### PR DESCRIPTION
**Pull request by barcelos-kevin**
Add my module to main branch and call it to main.py

It contains:
1. My Basic Information
2. Goals.
3. Simple To Do List Program

### CODE REVIEW

File: Exercise 12/ourpackages/barcelos.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                print("5. Have fun while coding :>")
+                input("\nPress Enter to continue. ")
+            case 3:
+                to_do_list()
+            case 0:
+                break
+            case _: 
+                print("Invalid Choice. Try Again.")
+                input("\nPress Enter to continue.")
+
```
Comment (veslino-marc) [inline Exercise 12/ourpackages/barcelos.py:60]: Wait, let me add my own comment in your own module - Marc

**Review by veslino-marc — CHANGES_REQUESTED** [review]

File: Exercise 12/ourpackages/barcelos.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                print(Fore.LIGHTYELLOW_EX + "1. Study Python.")
+                print("2. Pass this semester with good grades.")
+                print("3. Learn Data Analyst Skills.")
+                print("4. Get a part time job.")
+                print("5. Have fun while coding :>")
+                input("\nPress Enter to continue. ")
+            case 3:
+                to_do_list()
+            case 4:
+                print("Your code is clean and readable")
```
Comment (consultado-kirby) [inline Exercise 12/ourpackages/barcelos.py]: I will add my comment into your module. - Kirby

**Review by consultado-kirby — CHANGES_REQUESTED** [review]

File: Exercise 12/ourpackages/barcelos.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                print("5. Have fun while coding :>")
+                input("\nPress Enter to continue. ")
+            case 3:
+                to_do_list()
+            case 4:
+                print("Your code is clean and readable. - Marc")
+                input("\nPress Enter to continue. ")
+            case 5:
+                print("I like the idea of your codes. - Kirby")
+                input("\nPress Enter to continue. ")
```
Comment (andaya-gener) [inline Exercise 12/ourpackages/barcelos.py:51]: Let me add comment in your module. -Andaya

**Review by andaya-gener — CHANGES_REQUESTED** [review]

File: Exercise 12/ourpackages/barcelos.py (hunk 2 above)
Comment (barcelos-kevin) [reply Exercise 12/ourpackages/barcelos.py]: Ok you have done it :>

File: Exercise 12/ourpackages/barcelos.py (hunk 1 above)
Comment (barcelos-kevin) [reply Exercise 12/ourpackages/barcelos.py:60]: You have done it :>

File: Exercise 12/ourpackages/barcelos.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            case 6:
```
Comment (piadozo-edriane) [inline Exercise 12/ourpackages/barcelos.py:52]: I would like to add a comment to your module - Piadozo

**Review by piadozo-edriane — CHANGES_REQUESTED** [review]

File: Exercise 12/ourpackages/barcelos.py (hunk 3 above)
Comment (barcelos-kevin) [reply Exercise 12/ourpackages/barcelos.py:51]: You Have done it :>

File: Exercise 12/ourpackages/barcelos.py (hunk 4 above)
Comment (barcelos-kevin) [reply Exercise 12/ourpackages/barcelos.py:52]: _You have done it :>_

**Review by veslino-marc — APPROVED** [review]

**Review by consultado-kirby — APPROVED** [review]

**Review by piadozo-edriane — APPROVED** [review]
Nice

**Review by andaya-gener — APPROVED** [review]

**Pull request merged**